### PR TITLE
samples: matter: Align cluster changes in all Matter samples

### PR DESCRIPTION
**Pull request by ArekBalysNordic**
According to Matter Specs, for Matter 1.1.0.1 we need to set the Software Diagnostic Feature Map value to "0" and set the Basic Information Cluster Revision value to "1" in all our samples.

### CODE REVIEW

**Comment by kkasperczyk-no** [comment]
Shouldn't we align weather station and bridge as well?

File: applications/matter_bridge/src/bridge.zap [hunk 1]
```diff
@@ -3207,10 +3207,10 @@
               "side": "server",
               "type": "bitmap32",
               "included": 1,
-              "storageOption": "RAM",
+              "storageOption": "External",
```
Comment (kkasperczyk-no) [inline applications/matter_bridge/src/bridge.zap:3210]: Why did you change it from RAM to External? I'm not saying it's wrong, I just see FeatureMap is stored in RAM for all clusters, so I wonder why you decided to change it only for this one.

**Review by kkasperczyk-no — APPROVED** [review]

File: applications/matter_bridge/src/bridge.zap (hunk 1 above)
Comment (ArekBalysNordic) [reply applications/matter_bridge/src/bridge.zap:3210]: Recently, we have done it in this way for DoorLock sample, and AFAIR it worked properly so I aligned all samples to that change. @doublemis1 What do you think?

Comment (doublemis1) [reply applications/matter_bridge/src/bridge.zap:3210]: I am not aware about these solution, what suits best.
However, I agree with Arek that it should be aligned across all samples/clusters/endpoints

**Review by doublemis1 — APPROVED** [review]
